components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -332,6 +339,16 @@ class NewTabPage extends React.Component<Props, State> {
     this.props.saveShowBitcoinDotCom(!showBitcoinDotCom)
   }
 
+  toggleShowCryptoDotCom = () => {
+    const { showCryptoDotCom } = this.props.newTabData
+
+    if (!showCryptoDotCom) {
+      this.props.saveShowAddCard(true)
```

Comment:
I don't think this function should be responsible  for whether 'add card' is showing or not.

Reply:
@ryanml Any thoughts on a better placement for this function?

Reply:
I can rework this out of there as part of https://github.com/brave/brave-core/pull/6753 cc: @petemill